packages/material-ui/src/Grid/Grid.d.ts

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Breakpoint } from '../styles/createBreakpoints';
```

Comment:
Grid only supports the default breakpoints. However, this represents the breakpoints in the theme which can be different. This wasn't supported at the type level previously but is as of #20901.

Reply:
I think that we could leave a TODO. I sounds like something we want to solve in v5.

Reply:
What do you want to solve?

Reply:
I think that people should be able to use the custom breakpoints they add into the theme like that can use the sm, md, etc. We could consider it part of #11649.

Reply:
For instance `<Grid tablet={6} mobile={12} desktop={3} item />

Reply:
Sure, but that is unrelated to types.

Reply:
OK, sorry for the noise.